client/src/BuyerTenderList.js

```diff
@@ -144,10 +192,49 @@ const BuyerTenderList = () => {
 									{tender.no_of_bids_received}
 								</span>
 							</p>
+							{tender.status !== "Awarded" && tender.status !== "Closed" && (
+								<>
+									{editStatusId === tender.id ? (
```

Comment:
All of this is way too complicated.

I think the simplest strategy would be to only allow changing the status of a tender from the tender details page.

Reply:
do we have this?

Reply:
When the buyer clicks on the title of a `tender`, it should send them to the tender details page. There, they can see all the details of the tender, as well as the bids.